conan/internal/api/new/workspace.py

```diff
@@ -19,23 +19,23 @@
 
 include(FetchContent)
 
-function(add_project SUBFOLDER)
-    message(STATUS "Adding project ${SUBFOLDER}")
+function(add_project SUBFOLDER LIBRARY_NAME)
```

Comment:
Could we have this be default to SUBFOLDER if not specified? So that when users have the same path/lib name, no extra repeated argument would be necessary.

Maybe something like
```
if (NOT ARGN)
        set(LIBRARY ${SUBFOLDER})
    else()
        set(LIBRARY ${ARGN})
    endif()
```